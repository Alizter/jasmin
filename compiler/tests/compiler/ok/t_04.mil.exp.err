Parsing tests/compiler/ok/t_04.mil as MIL

Processed file tests/compiler/ok/t_04.mil
