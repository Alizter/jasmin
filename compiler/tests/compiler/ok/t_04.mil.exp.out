renumbering identifiers in all function(s)
   REMOVED_TIME
>> input:
fn foo1(x.1 : stack u64, y.2 : stack u64, z.3 : reg bool) -> stack u64 {
  w.4 : stack u64
  (w.4 : stack u64) = (x.1 : stack u64);
  (w.4 : stack u64) += (x.1 : stack u64);
  if (w.4 : stack u64 = 5) {
    (z.3 : reg bool), (x.1 : stack u64) += (w.4 : stack u64);
    (z.3 : reg bool), (x.1 : stack u64) += (y.2 : stack u64);
  }
  return (x.1 : stack u64);
}

fn foo2(x.5 : stack u64, y.6 : stack u64, z.7 : reg bool) -> stack u64 {
  w.8 : stack u64
  (w.8 : stack u64) = (x.5 : stack u64);
  (w.8 : stack u64) += (x.5 : stack u64);
  if (w.8 : stack u64 = 5) {
    (z.7 : reg bool), (x.5 : stack u64) += (w.8 : stack u64);
    (z.7 : reg bool), (x.5 : stack u64) += (y.6 : stack u64);
  }
  return (x.5 : stack u64);
}


total transformation time: REMOVED_TIME

total time (with save/print): REMOVED_TIME
